instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integrationtests/ActivityProfiling.kt

```diff
@@ -4,15 +4,15 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.datadog.android.log.Logger
 import com.datadog.android.sdk.integration.R
-import com.datadog.android.sdk.integration.Runtime
+import com.datadog.android.sdk.integration.RuntimeConfig
 
 internal class ActivityProfiling : AppCompatActivity() {
 
     lateinit var logger: Logger
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        logger = Runtime.logger()
+        logger = RuntimeConfig.logger()
```

Comment:
oh....actually this is better :). I had to use reflection to make it work without the `setEndpoint` method but I think this thing is better. I will update this in my next PR.